tests/explorer.api.tests/ApiTests.cs

```diff
@@ -106,11 +94,45 @@ public async Task SuccessWithContents()
             });
         }
 
-        [Theory]
-        [MemberData(nameof(SuccessWithResultTestData))]
-        public async Task SuccessWithResult(Models.ExploreParams pdata)
+        [Fact]
+        public async Task SuccessWithResultNoColumns() => await SuccessWithResult(
+            new ExploreParams { DataSource = "gda_banking", Table = "loans", Columns = ImmutableArray<string>.Empty, });
```

Comment:
What are `[Theory]` and `[Fact]` annotations? Are they for a test framework? How does this work?

Reply:
Yes, `[Theroy]` and `[Fact]` are part of `Xunit` testing framework. These are used to mark the methods the are unit tests. `[Theory]` marks a unit test method that receives additional parameters, and it will be executed multiple times, once for each instance of the parameters specified, e.g. in this case the parameters are specified using the `[MemberData]` annotation.